blocks/embed/block.json

```diff
@@ -85,6 +85,8 @@
     }
   },
   "supports": {
-    "align": ["wide", "full", "left", "right", "center"]
-  }
+    "align": ["wide", "full", "left", "right", "center"],
+    "interactivity": true
+  },
+  "viewScriptModule": "file:../../packages/stories-block-view/src/index.js"
```

Comment:
This should point to the webpack-built JavaScript file in the assets folder.

The final plugin installed by users does not have a `packages` folder.